src/vellum/workflows/nodes/core/map_node/node.py

```diff
@@ -53,7 +55,15 @@ def run(self) -> Outputs:
         fulfilled_iterations: List[bool] = []
         for index, item in enumerate(self.items):
             fulfilled_iterations.append(False)
-            thread = Thread(target=self._run_subworkflow, kwargs={"item": item, "index": index})
+            parent_context = get_parent_context() or self._context.parent_context
```

Comment:
[oos] If this is `get_execution_context().parent_context`, we could start to move the other items in `self._context` into the execution context and remove the `_context` param from the base node init